server/controllers/article.js

```diff
@@ -116,35 +107,60 @@ class ArticleController {
           attributes: ['id', 'email', 'inAppNotification', 'emailNotification']
         }
       ]
-    }))
-      .forEach((follower) => {
-        follower = follower.follower.dataValues;
-
-        if (follower.inAppNotification) {
-          // Insert notification
-          UserNotification.create({
-            userId: follower.id,
-            notificationId: notification.id
-          });
+    });
+    // No need to create notification if user has no followers.
+    if (followers.length <= 0) {
+      return;
+    }
 
-          // Send push notification to each user's channel.
-          const { message, url, type } = notification;
-          HelperUtils.pusher(`channel-${follower.id}`, 'notification', {
-            message,
-            url,
-            type
-          });
-        }
+    // Create notification
+    const notification = await Notification.create({
+      message: `${author.firstname} ${author.lastname} just published an article`,
+      metaId: article.id,
+      type: 'article.published',
+      url: `/${article.slug}`
+    });
 
-        if (follower.emailNotification) {
-          // Send email notification
-          HelperUtils.sendMail(follower.email,
-            'Authors Haven <no-reply@authorshaven.com>',
-            notification.message,
-            notification.message,
-            articleNotificationMarkup(`${author.firstname} ${author.lastname}`, article.title, article.description, notification.url));
-        }
-      });
+    const dbRows = []; // Database rows for users that opted for in-app notifications.
+    const emails = ['danielaustin718@gmail.com']; // Email addresses of users that want email notifications.
```

Comment:
Why do you need to hard code your email address; kindly confirm

Reply:
I added it so I could see what the email would look like. I forgot to remove it.